ee/modules/380-metallb/images/artifact/patches/001-l2-speker-preffered-node.patch

```diff
@@ -0,0 +1,38 @@
+diff --git a/speaker/layer2_controller.go b/speaker/layer2_controller.go
```

Comment:
NB for reviewers! For now, the new inlet rely only on nodeSelector to detect frontend-nodes, not tolerations. Not sure we need them.